test/routes/signup.js

```diff
@@ -0,0 +1,127 @@
+var request = require('supertest'),
+    should = require('should'),
+    mongoose = require('mongoose'),
+    config = require('../../config/env/all');
+
+describe('Signup Integration test', function () {
+    this.timeout(140008);
+
+    var url = 'http://localhost:' + process.env.PORT;
```

Comment:
@andela-akolawole 
Please use variable name that very descriptive, baseUrl is very better here